service/src/main/java/bio/terra/catalog/common/StorageSystem.java

```diff
@@ -1,6 +1,30 @@
 package bio.terra.catalog.common;
 
 public enum StorageSystem {
-  TERRA_WORKSPACE,
-  TERRA_DATA_REPO
+  TERRA_WORKSPACE(bio.terra.catalog.model.StorageSystem.WKS.name()),
+  TERRA_DATA_REPO(bio.terra.catalog.model.StorageSystem.TDR.name()),
+  EXTERNAL(bio.terra.catalog.model.StorageSystem.EXT.name());
```

Comment:
Yes, I mean using the fully qualified name. I don't think we should use different names for each, but it's not clear to me we can't do something like this:

```
import bio.terra.catalog.model.StorageSystem;

...

TERRA_WORKSPACE(StorageSystem.WKS.name()),
TERRA_DATA_REPO(StorageSystem.TDR.name()),
EXTERNAL(StorageSystem.EXT.name());
```

Reply:
That import isn't allowed in Java because it introduces the name `StorageSystem` into the current scope. When it gets to the line `public enum StorageSystem` you get an error that the name already exists. The code could static import the model enum contents though:

```
import static bio.terra.catalog.model.StorageSystem.*;

public enum StorageSystem {
  TERRA_WORKSPACE(WKS.name()),
  TERRA_DATA_REPO(TDR.name()),
  EXTERNAL(EXT.name());
```

But I don't usually like static imports since they make it harder for a reader to tell where a referenced name was defined.